fsharp-backend/src/LibExecution/StdLib/LibNoModule.fs

```diff
@@ -140,22 +140,37 @@ let fns : List<BuiltInFn> =
 //   ; sqlSpec = NotYetImplementedTODO
 // ; previewable = Pure
 //   ; deprecated = ReplacedBy(fn "" "" 0) }
-// ; { name = fn "AWS" "urlencode" 0
-//   ; parameters = [Param.make "str" TStr ""]
-//   ; returnType = TStr
-//   ; description = "Url encode a string per AWS' requirements"
-//   ; fn =
-//         (function
-//         | _, [DStr str] ->
-//             str
-//             |> Unicode_string.to_string
-//             |> Stdlib_util.AWS.url_encode
-//             |> DStr
-//         | _ ->
-//             incorrectArgs ())
-//   ; sqlSpec = NotYetImplementedTODO
-// ; previewable = Pure
-//   ; deprecated = NotDeprecated }
+    { name = fn "AWS" "urlencode" 0
+      parameters = [Param.make "str" TStr ""]
+      returnType = TStr
+      description = "Url encode a string per AWS' requirements"
+      fn =
+          (function
+          | _, [ DStr s ] -> 
+              let n = String.length s
```

Comment:
Can you adapt whatever comments are relevant from https://github.com/darklang/dark/blob/main/backend/libexecution/stdlib_util.ml#L5 and add them here?